src/Rules/Resources/ErrorMessages.resx

```diff
@@ -150,6 +150,9 @@
   <data name="NoControlTypeEntryFound" xml:space="preserve">
     <value>No control type entry was found in dictionary</value>
   </data>
+  <data name="NoElementFound" xml:space="preserve">
+    <value>Based on the Rule.Condition match, there should have been at least one element found in the PassesTest method of {0}</value>
```

Comment:
Will this be exposed to users? It seems a little too obscure to really help them